src/components/video-with-device-frame.js

```diff
@@ -20,29 +20,31 @@ class VideoWithDeviceFrame extends React.Component {
     } else {
       imageId = 'pixel2FrameHorizontal';
     }
-    const videoClasses = classnames('block mx-auto  ', {
-      'pt90 px12 w300 pb120':
+    const videoClasses = classnames('block mx-auto', {
+      'pt90 px9 w300 pb120':
         this.props.rotation === 'vertical' && this.props.device === 'pixel-2',
       'w600-mxl px84-mxl pt15-mxl w480-ml px60-ml pt12-ml w360 px48 pt6 pb60':
         this.props.rotation === 'horizontal' && this.props.device === 'pixel-2'
     });
-    const frameClasses = classnames('mt0', {
+    const frameClasses = classnames('mt0 mx-auto', {
       'pt12 w360': this.props.rotation === 'vertical',
       'w600-mxl w480-ml w360': this.props.rotation === 'horizontal'
     });
 
     return (
       <div className="relative">
         <div className="top left right align-center absolute">
-          <AppropriateImage
-            className={frameClasses}
-            style={frameHorizontalCustomStyles}
-            imageId={imageId}
-          />
+          <div className={frameClasses}>
+            <AppropriateImage
+              style={frameHorizontalCustomStyles}
+              imageId={imageId}
```

Comment:
Would you mind also adding an empty alt attribute to this component? We're getting dinged in Lighthouse (accessibility and SEO) for not having an alt attribute on this image. We can use an empty string since it's decorative.

```suggestion
              imageId={imageId}
              alt=""
```